scripts/fires/manifest.json

```diff
@@ -1,15 +0,0 @@
-{
```

Comment:
why has this been removed - are we not going to do autorefreshes anymore?

Reply:
We will do it in a subsequent PR once this is submitted